### PR TITLE
Use manifest.json in build and tests scripts

### PR DESCRIPTION
**Pull request by natemcmaster**
Resolves https://github.com/aspnet/aspnet-docker/issues/252

### CODE REVIEW

File: test/test.ps1 [hunk 1]
```diff
@@ -181,3 +156,46 @@ $files | % {
...
+    $docker_test_file = "Dockerfile.test.nanoserver"
+}
+else {
+    $container_root = "/"
+    $host_port = "5000"
+    $rid="debian.8-x64"
+    $docker_test_file = "Dockerfile.test.linux"
+}
+
+$platform = docker version -f "{{ .Server.Os }}"
```
Comment (MichaelSimons) [inline test/test.ps1]: Looks like this is defined multiple times.

File: test/test.ps1 [hunk 2]
```diff
@@ -96,7 +71,7 @@ $files | % {
     try {
         $framework ="netcoreapp${version}"
         exec {
-            (Get-Content (Join-Path $PSScriptRoot -ChildPath "Dockerfile.test.$image_os")).
+            (Get-Content (Join-Path $PSScriptRoot -ChildPath $docker_test_file)).
```
Comment (MichaelSimons) [inline test/test.ps1]: One of the things I dislike about Powershell scripting is that variables have no scope.  I see the docker_test_file variable isn't passed in yet version, sdk_tag, and runtime_tag are but wouldn't strictly have to be.  Curious if there is a recommended best practice to follow in this situation.  At a minimum it seems like consistency should be used across all variables.

File: test/test.ps1 [hunk 3]
```diff
@@ -181,3 +156,46 @@ $files | % {
...
+}
+
+$platform = docker version -f "{{ .Server.Os }}"
+$manifest = Get-Content (Join-Paths $PSScriptRoot ('..', 'manifest.json')) | ConvertFrom-Json
+
+# Main
+$manifest.repos | % {
+    $repo = $_
+    $repoName = $repo.name -replace 'microsoft/',"$RootImageName/"
+    Write-Host "Building ${repoName}"
```
Comment (MichaelSimons) [inline test/test.ps1]: Should this be changed to 'Testing'?

File: test/test.ps1 (hunk 2 above)
Comment (natemcmaster) [inline test/test.ps1]: We can be explicit about scope
```ps1
$global:docker_test_file
```
Though I haven't used this much either.

File: test/test.ps1 (hunk 3 above)
Comment (natemcmaster) [inline test/test.ps1]: Updated

File: test/test.ps1 (hunk 1 above)
Comment (natemcmaster) [inline test/test.ps1]: Fixed :)

**Pull request merged**